src/app/admin/volunteers/volunteers-list/volunteers-list.component.ts

```diff
@@ -87,7 +87,7 @@ export class VolunteersListComponent implements OnInit {
       {
         name: 'Sector',
         value: 'zone_address',
-        array: of(KIV_ZONES),
+        array: of(zones),
```

Comment:
This I think needs to be reverted, because `KIV_ZONES` has a different structure than `zones`
And is passed somewhere into a filter component that uses `KIV_ZONES` structure